bootstrap/src/lib/cache-warmer/store.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  AnyAction,
+  applyMiddleware,
+  compose,
+  createStore,
+  Dispatch,
+  Middleware,
+  PreloadedState,
+  Reducer,
+} from 'redux'
+import logger from 'redux-logger'
+import { createEpicMiddleware } from 'redux-observable'
+import { composeWithDevTools } from 'remote-redux-devtools'
+import { Config, get } from './config'
+import { RootState } from './reducer'
+export interface EpicDependencies {
+  config: Config
+}
+
+export function configureStore(rootReducer: Reducer, preloadedState: PreloadedState<any> = {}) {
+  const epicMiddleware = createEpicMiddleware<any, any, RootState, EpicDependencies>({
+    dependencies: { config: get() },
+  })
+
+  const middlewares: Middleware<unknown, any, Dispatch<AnyAction>>[] = [epicMiddleware]
+  if (process.env.DEBUG || process.env.NODE_ENV === 'development') {
```

Comment:
We should consider enabling the logger here for `LOG_LEVEL === 'debug'`.

Reply:
Good idea, will do